sdk/purview/Azure.Analytics.Purview.Catalog/src/Generated/PurviewCatalogClient.cs

```diff
@@ -64,6 +64,68 @@ public PurviewCatalogClient(Uri endpoint, TokenCredential credential, PurviewCat
         /// <param name="content"> The content to send as the body of the request. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="content"/> is null. </exception>
+        /// <example>
+        /// This sample shows how to call SearchAsync and parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new PurviewCatalogClient(endpoint, credential);
+        /// 
+        /// var data = new {};
+        /// 
+        /// Response response = await client.SearchAsync(RequestContent.Create(data));
+        /// JsonElement result = JsonDocument.Parse(GetContentFromResponse(response)).RootElement;
+        /// Console.WriteLine(result.ToString());
+        /// ]]></code>
+        /// This sample shows how to call SearchAsync with all request content, and how to parse the result.
+        /// <code><![CDATA[
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("<https://my-account-name.azure.com>");
+        /// var client = new PurviewCatalogClient(endpoint, credential);
+        /// 
+        /// var data = new {
+        ///     keywords = "<SearchRequestKeywords>",
+        ///     offset = 1234,
+        ///     limit = 1234,
+        ///     filter = new {},
+        ///     facets = new[] {
+        ///         new {
+        ///             count = 1234,
+        ///             facet = "<SearchFacetItemFacet>",
+        ///             sort = new {},
+        ///         }
+        ///     },
+        ///     taxonomySetting = new {
+        ///         assetTypes = new[] {
+        ///             "<SearchRequestTaxonomySettingAssetTypesItem>"
+        ///         },
+        ///         facet = new {
+        ///             count = 1234,
+        ///             facet = "<SearchFacetItemFacet>",
+        ///             sort = new {},
+        ///         },
+        ///     },
+        /// };
+        /// 
+        /// Response response = await client.SearchAsync(RequestContent.Create(data));
```

Comment:
Would it be possible to add a carriage return after the client call to separate this from parsing the response?

Reply:
Thanks! :)